docs/application/native/guides/multimedia/media-conversions.md

```diff
@@ -205,7 +205,7 @@ To run the media codec loop:
 
      The eos callback is invoked if the eos packet is decoded or encoded and the eos callback is registered with the `mediacodec_set_eos_cb()` function.
 
-   > **Note**  
+   > **Note**
```

Comment:
Hello, @neoiomok 
To set this line as a title, we need a line break for **Note**.
Would you please add empty line with > as below?

```
   > **Note**
   >
   > You must set the flags before calling the `mediacodec_process_input()` function.
```